internal/fusefrontend_reverse/rfs.go

```diff
@@ -379,15 +341,15 @@ func (rfs *ReverseFS) OpenDir(cipherPath string, context *fuse.Context) ([]fuse.
 // cDir is the relative ciphertext path to the directory these entries are
 // from.
 func (rfs *ReverseFS) excludeDirEntries(cDir string, entries []fuse.DirEntry) (filtered []fuse.DirEntry) {
-	if rfs.cExclude == nil {
+	if rfs.excluder == nil {
 		return entries
 	}
 	filtered = make([]fuse.DirEntry, 0, len(entries))
 	for _, entry := range entries {
 		// filepath.Join handles the case of cipherPath="" correctly:
 		// Join("", "foo") -> "foo". This does not: cipherPath + "/" + name"
 		p := filepath.Join(cDir, entry.Name)
-		if rfs.isExcluded(p) {
+		if excluded, _, _ := rfs.isExcludedCipher(p); excluded {
```

Comment:
This is a bit inefficient: The paths are first encrypted and then decrypted just to check if they are excluded.

Reply:
This should be fixed.